paws_utils.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 const moment = require('moment');
+const NEXT_POLL_INTERVAL_DELAY = process.env.paws_poll_interval_delay && process.env.paws_poll_interval_delay <= 900 ? process.env.paws_poll_interval_delay : 600;
```

Comment:
I think we need to move this var inside `calcNextCollectionInterval()` and default its value to `pollInterval` and not `600`

Reply:
The default  value of pollInterval in all the collector is 60seconds. If we used this it will just keep 1 min interval ,it will be the same as existing logic.